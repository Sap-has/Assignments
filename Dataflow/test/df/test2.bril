@main {
  x: int = const 10;
  y: int = const 5;
  cond: bool = gt x y;
  br cond .then .end;
.then:
  z: int = add x y;
  jmp .end;
.end:
  print x;
}