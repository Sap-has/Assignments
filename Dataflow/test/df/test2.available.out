b1:
  in:  âˆ…
  out: gt x y
then:
  in:  gt x y
  out: add x y, gt x y
end:
  in:  gt x y
  out: gt x y
